resolwe/flow/filters.py

```diff
@@ -158,7 +161,24 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
 
-class BaseResolweFilter(CheckQueryParamsMixin, filters.FilterSet):
+class ResolweFilterMetaclass(FilterSetMetaclass):
+    """Metaclass for Resolwe filters."""
+
```

Comment:
I believe this code deserves some comment. At least explain that our extensions for the given class are injected as argument attrs in FilterSetMetaclass and that they are loaded in there.

This solution is very nice, but it depends on the actual implementation of django_filters package, not just its API. Maybe it would be nice to mention that in a comment for easier debugging if/when the implementation of django_filters changes.